database/database.go

```diff
@@ -192,6 +190,36 @@ func (db *DB) CreateBlockedSkylink(ctx context.Context, skylink *BlockedSkylink)
 	return nil
 }
 
+// CreateBlockedSkylinkBulk creates new blocked skylinks in bulk. It returns the
+// number of created entries.
+func (db *DB) CreateBlockedSkylinkBulk(ctx context.Context, skylinks []BlockedSkylink) (int, error) {
```

Comment:
the naming is off here but I'm tackling that in another PR that runs through the whole blocker and gets rid of the term "skylink" where its not appropriate any longer